JaxTeller718/JaxTeller718v1.0Modlets
# Fix Increased Zombie Spawns for 1.2

### What was done
* Fixed "Duplicate id hash" error preventing startup 

### How to test

1. Add to mods per repo instructions
2. Boot w/o EAC
3. Verify you can launch game

### Notes

- Not actually sure if this **retains functionality**, but sure as heck lets the game boot.
